BritneyS/GetFed
# Custom textfields

## What you did :question:
- Styled textfields in Food Entry screen to only include bottom border
- Added more styling to Food Entry labels
**UPDATE 12-10-18: Added Podfile and Info.plist**

## How you did it :white_check_mark:
- Added textfield border layer in custom subclass
- Added dark purple and dark green as Color Assets to Color.xcassets folder
- Added cases for new color assets in enum ColorID
- Styled Food Entry labels in Interface Builder


## How to test it :microscope:
- Run the app
- Tap 'Food Entry'
- Note that the labels should appear dark purple and bolded, and the textfields should appear as dark green bottom borders


## Screenshots (if applicable) :camera:
![simulator screen shot - iphone xr - 2018-12-07 at 15 41 24](https://user-images.githubusercontent.com/8409475/49671882-59b54f80-fa37-11e8-81ac-3f4bb4eaf185.png)
